lib/rust/ensogl/example/render-profile-flamegraph/src/lib.rs

```diff
@@ -35,25 +38,33 @@ use ensogl_flame_graph as flame_graph;
 #[allow(dead_code)]
 pub async fn entry_point_render_profile_flamegraph() {
     use ensogl_core::display::object::ObjectOps;
-    let app = application::Application::new("root");
+    let app = &application::Application::new("root");
     let world = &app.display;
     let scene = &world.default_scene;
+    let network = app.frp.network();
     let navigator = navigator::Navigator::new(scene, &scene.camera());
     init_theme(scene);
     let data = get_data().await;
     let profile: profiler_data::Profile<profiler_data::OpaqueMetadata> = data.parse().unwrap();
     let mut builder = profiler_flame_graph::FlamegraphBuilder::default();
     builder.add_profile(&profile);
-    let flame_graph = flame_graph::FlameGraph::from_data(builder.into(), &app);
+    let flame_graph = flame_graph::FlameGraph::from_data(builder.into(), app);
     scene.add_child(&flame_graph);
     scene.layers.main.add_exclusive(&flame_graph);
     world.keep_alive_forever();
+    let tooltip = ensogl_tooltip::Tooltip::new(app);
```

Comment:
Well, architecturally, this is part of the demo scene, as it receives its FRP events from the application. We could add it as part of the `FlameGraph`, but if we ever use the `FlameGraph` in some other context where another tooltip is used, this will lead to problems.